api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/remoteOperations.md

```diff
@@ -16,6 +16,8 @@ If you enable this property, the PivotGrid sends several requests to load data.
         take: 20
     }
 
+[note]If you use [local stores](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) in the PivotGridDataSource, disable the `remoteOperations` property to ensure that usage scenarios works properly.
```

Comment:
What stores are considered local? Provide examples

Reply:
ArrayStore and LocalStore